src/components/common/EnhancedTable/index.tsx

```diff
@@ -150,13 +149,15 @@ function EnhancedTable({ rows, headCells, variant }: EnhancedTableProps) {
                   selected={row.selected}
                   className={row.collapsed ? css.collapsedRow : undefined}
                 >
+                  {CellCard && <CellCard row={row} className={css.cellCard} />}
```

Comment:
Not sure if we need to render a different component for mobile. It should be possible to break the table cells via `display` into a row based layout.

Reply:
I wanted to achieve displaying a custom Card that can be passed to the `<EnhancedTable>` for mobile views, as we've already had some issues and designs that suggested going in that direction.
I've considered keeping the table cells as well but the reason I've decided for this approach was the flexibility it gives.
Do you find the other approach more suitable?

Reply:
I would try to avoid rendering the same content twice if avoidable e.g. if we can achieve layout changes with css

Reply:
I've removed the Card component and made the table accept a `mobileVariant`